pom.xml

```diff
@@ -18,11 +18,11 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<version.wildfly.maven.plugin>1.0.2.Final</version.wildfly.maven.plugin>
 		<version.jboss.bom>8.2.1.Final</version.jboss.bom>
-		<version.compiler.plugin>3.1</version.compiler.plugin>
+		<version.compiler.plugin>3.7.0</version.compiler.plugin>
 		<version.surefire.plugin>2.16</version.surefire.plugin>
 		<version.war.plugin>2.5</version.war.plugin>
-		<maven.compiler.target>1.7</maven.compiler.target>
-		<maven.compiler.source>1.7</maven.compiler.source>
+		<maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
I understand this, since you are using Objects.hashCode() for calculating hash code, but... we need to stick with 1.7 for now, since R serve will be broken if using 1.8